src/main/java/dune/Dune.java

```diff
@@ -0,0 +1,48 @@
+package dune;
+
+import dune.task.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * The main class for the program.
+ */
+public class Dune {
+
+
+    private TaskList tasks;
+
+    private Storage storage;
+
+    private Parser p;
```

Comment:
Perhaps use a more detailed name here (e.g. = parser)? Since it is a global variable for the class which will be used throughout the class.